src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx

```diff
@@ -118,9 +118,17 @@ class PaymentInformationEditModal extends React.Component {
         onClose={this.props.onClose}
       >
         {!!this.props.responseError && (
-          <PaymentInformationEditModalError
-            responseError={this.props.responseError}
-          />
+          <div id="errors" role="alert" aria-atomic="true">
+            <div className="usa-alert usa-alert-error">
```

Comment:
The `PaymentInformationEditModalError` component renders out the DOM starting with this `div` right here, so you don't need to add it to the markup. We just need to wrap the existing `PaymentInformationEditModalError` component in the `div` you added at line 121.

Reply:
Aren't I doing that? Or maybe I am missing something?

Reply:
No you're wrapped the `PaymentInformationEditModalError` in a bunch of extra markup that isn't needed because that extra markup is exactly what the `PaymentInformationEditModalError` component renders. That component simply renders out an `AlertBox` component and you can find the source code of the `AlertBox` component here: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/47dcd54d0ac7cc8917452a4328d108630cd79db7/packages/formation-react/src/components/AlertBox/AlertBox.jsx. So you're effectively wrapping an AlertBox in another AlertBox.

And actually reviewing Trevor's original suggestion for the fix, the wrapper div, `<div id="errors" role="alert" aria-atomic="true">` should live _outside_ of the conditional logic that renders the `PaymentInformationEditModalError` component if `props.responseError` is set.

Reply:
Oh ok. Dang sorry about the long way round explanation.

Reply:
Ok I just push that code.